core/pom.xml

```diff
@@ -38,6 +38,11 @@
       <optional>true</optional>
     </dependency>
 
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
```

Comment:
sorry mate, this is not going to happen.

feign core shall have no dependencies

Reply:
Alternate would be use regex to validate the json string without any external dependencies. I can already see a feign-json library which uses org.json to encode/decode json. I assume its the right place to add the json literal validation. But we would still need to add the feign-json library into feign-core to use the implementation.

Reply:
all modules have multiple dependecies.... `feign-core` does not